### ISSUE TITLE
Missing freetype::readOptions in freetype_fallback

### ISSUE THREAD

**Issue by kannode** [vsg-dev/vsgXchange#140]
When building on Windows, MSVC complains about a missing placeholder function, ```freetype::readOptions.```

```
2>vsgXchanged.lib(freetype_fallback.obj) : error LNK2001: unresolved external symbol "public: virtual bool __cdecl vsgXchange::freetype::readOptions(class vsg::Options &,class vsg::CommandLine &)const " (?readOptions@freetype@vsgXchange@@UEBA_NAEAVOptions@vsg@@AEAVCommandLine@4@@Z)

3>D:\vsg\vsgXchange\build\bin\Debug\vsgconvd.exe : fatal error LNK1120: 1 unresolved externals
3>Done building project "vsgconv.vcxproj" -- FAILED.
```

I expect adding to freetype_fallback.cpp should fix it.
```
bool freetype::readOptions(vsg::Options&, vsg::CommandLine&) const
{
    return false;
}
```

**Comment by robertosfield** [comment]
Thanks,  Adding in the missing readOptions() is appropriate.  When I get to my dev machine I'll add it and check it in.

**Comment by robertosfield** [comment]
Fix now checked in: afa3ca680add43e71d5e5348d3fa92a131f11841
